config/assets/default.js

```diff
@@ -35,6 +35,9 @@ module.exports = {
       'modules/*/client/*.js',
       'modules/*/client/**/*.js'
     ],
+    img: [
+      'modules/**/**/img/**/*'
```

Comment:
Handle whichever ones the png library supports? Anything else and we may have an error, so we can just avoid that by using the multiple term matching syntax, which I think is `(abc|xyz|def)`, but verify that.